lib/spack/spack/build_systems/python.py

```diff
@@ -403,3 +407,62 @@ def import_module_test(self):
 
     # Check that self.prefix is there after installation
     run_after('install')(PackageBase.sanity_check_prefix)
+
+    def view_file_conflicts(self, view, merge_map):
```

Comment:
This should get at least a general comment on what is happening here, I think I got the gist of it, but for someone just trying to figure out what is happening this is pretty hard to read

Reply:
I added a header comment mentioning the reason for the override. Let me know if it could use additional clarification.

Reply:
I think this needs documentation at the package base class level too (or in a mixin if package is getting too heavily loaded)